src/app/core/models/user.model.ts

```diff
@@ -1,6 +1,13 @@
 export interface User {
-    body: string,
     id:number,
-    title:string,
-    userId: number
+    name: string,
+    email: string,
+    mobile?: string,
+    city?: string,
+    age?: string,
+    memberOf?: string,
+    skills?: string,
+    aboutMe?: string,
+    organizations?: string,
```

Comment:
the same as about skills and organization

Reply:
Organizations and skills let leave as it is